lib/app.rb

```diff
@@ -0,0 +1,32 @@
+require_relative './game'
+require_relative './board'
+require_relative './player'
+require 'sinatra'
+require 'json'
+require 'pry'
+
+enable :sessions
+
+  get '/' do 
+    erb :index
+  end
+
+  post '/players' do
+    @player1 = Player.new(params[:player1_name], 'M')
+    @player2 = Player.new(params[:player2_name], 'T')
+    session[:player1] = @player1
+    session[:player2] = @player2
+    erb :players
+  end
+
+  get '/game' do
+    session[:game] = Game.new(session[:player1], session[:player2])
+    @game = session[:game]
+    erb :game
+  end
+
+  post '/set_cell' do
```

Comment:
this looks exactly like app.rb  why do we need two files that do the same thing?